css/pseudoelements/demos/first-letter/index.html

```diff
@@ -9,28 +9,48 @@
   <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap">
   <style>
-    body {
+    *, *::before, *::after {
       margin: 0;
-      padding: 5%;
-      background-color: #282a2e;
-      color: #ffffff;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    body {
+      min-height: 100vh;
+      padding: 50px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background-color: #18191c;
       font-family: "Roboto", sans-serif;
-      overflow: hidden;
+      font-size: 18px;
     }
 
     section {
-      max-width: 80%;
-      margin: 0 auto;
-      font-size: 18px;
-      line-height: 27px;
+      width: 65%;
+      padding: 55px 40px;
+      background-color: #2E9AFF;
+      color: #000000;
+      line-height: 1.4;
     }
 
     .text:first-letter {
-      color: #ed4242;
+      color: #F498AD;
       font-family: "Roboto Slab", serif;
       font-size: 52px;
       line-height: 45px;
```

Comment:
```suggestion
      line-height: 0.8;
```

Тут точно высота строки меньше размера текста?

Reply:
Да, потому что это буквица :) Здесь никогда не будет второй строки такого же текста, чтобы возникло наложение строки на строку. Но при этом большая высока строки у буквицы затрагивает всю первую строку и получается вот так:

[![Image from Gyazo](https://i.gyazo.com/05d2442007b19b204f21f4669ba435ab.png)](https://gyazo.com/05d2442007b19b204f21f4669ba435ab)

Так что я решила, что подрезать line-height в этом случае будет достаточно безопасно.